### PR TITLE
dckw-617: update wallet sdk docs

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

**Review by AgneCaunt — DISMISSED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+## Installation
+
+To start, you need to install the necessary packages for the wallet SDK and the data store. Open your terminal and run the following command:
+
+```bash
+yarn add @docknetwork/wallet-sdk-core @docknetwork/wallet-sdk-data-store-typeorm
+```
+
+The `@docknetwork/wallet-sdk-core` provides the core wallet functionality, while `@docknetwork/wallet-sdk-data-store-typeorm` handles data persistence using a local SQLite database.
```
Comment (mike-parkhill) [inline docs/getting-started.md:13]: should we talk about using the cloud wallet storage somewhere here?

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+const presentation = await controller.createPresentation(); // Create a presentation based on the selected credentials
+
+const verificationResponse = await controller.submitPresentation(presentation); // Submit the presentation for verification
+
+console.log(verificationResponse);
+```
+
+In this example, the verification process starts by fetching a proof request template. After selecting the credentials and specifying which attributes to reveal, the wallet creates a verifiable presentation. The presentation is then submitted, and the verifier responds with a verification result.
+
+### Conclusion
```
Comment (mike-parkhill) [inline docs/getting-started.md:121]: not sure we need a Conclusion section - I don't feel it's adding much value

File: docs/getting-started.md (hunk 1 above)
Comment (maycon-mello) [reply docs/getting-started.md:13]: I've added that to line 33

Comment (maycon-mello) [reply docs/getting-started.md:13]: https://github.com/docknetwork/react-native-sdk/pull/297/files#diff-31bcba2ccafa41d46fbbd6d1219f7f1e3b1fb3cad9faa8e4dc521bbb579dd7b3R33

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**